src/rcPresenceFail/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Danger message failure.
+ * @constant
+ * @type {String}
+ * @default
+ */
+const MESSAGE = `<b>Cannot merge release candidate!</b> - <i>
+It seems you forgot to remove "-rc" from your package.json file.
```

Comment:
Pull requests **should** have RC (sometimes)
```suggestion
Please remove "-rc" from your package.json file before merging to main branch.
```

Reply:
👍